.dangerfile/readme_tests.rb

```diff
@@ -355,7 +355,7 @@ def readme_invalid_credentials?(file, file_lines)
           footnote_symbols["¶"] = true if line.strip.end_with?("\u00b6")
 
           permission_action = line.split("  - ")[1]
-          if permission_action.nil? || !permission_action.match?(azure_perm_tester)
+          if permission_action.nil? || !permission_action.match?(azure_perm_tester) && !permission_action.include?("Storage Blob Data")
```

Comment:
We should have the permission rather than the role assigned

I can track down this permission/s

Will likely be `Microsoft.Storage/storageAccounts/blobServices/containers/blobs/read`

And potentially the one for reading containers